drivers/sensor/grove/Kconfig

```diff
@@ -8,7 +8,7 @@
 
 config GROVE_LIGHT_SENSOR
 	bool "Enable the Seeed Grove Light Sensor"
-	depends on ADC && NEWLIB_LIBC
+	depends on ADC && !MINIMAL_LIBC
```

Comment:
why don't these become `FULL`?

Reply:
Not clear what the exact question is, so 2 answers:

1. There is no `FULL_LIBC` only a `REQUIRES_FULL_LIBC` to indicate that a given config requires a full featured libc. The exact libc implementation can be set by the user (`newlib` or `external`)
2. The original code used `depends on` and not `select`, so just kept it backward compatible

Reply:
I was asking why isn't this 'depends on ADC && REQUIRES_FULL_LIBC`?

Reply:
`REQUIRES_FULL_LIBC` is a hidden attribute used to change the _default_ value of the `choice LIBC_IMPLEMENTATION` and can only be set via a `select` in another attribute (or via a defconfig).
As such `REQUIRES_FULL_LIBC` can not be used in `depends on` of attributes as it is not guaranteed it will be set even when _full_ libc is selected.